en/docs/design/api-security/api-authentication/secure-apis-using-basic-authentication.md

```diff
@@ -29,6 +29,8 @@ anyone of mandatory will skip the authentication.
  
 Note : If OAuth2/Basic Auth is set as mandatory, the request needs to be authenticated using only one of them. If OAuth2 failed only, the Basic Authentication will be applied.
 
+!!! note
```

Comment:
Shall we change this note slightly to give the idea properly. The APIs with basic authentication are not bound to an application since they do not require a subscription. Therefore, they do not support application level throttling. However, API level throttling is applicable for them.